internal/slicer/slicer.go

```diff
@@ -246,6 +250,17 @@ func Run(options *RunOptions) error {
 		}
 	}
 
+	for _, path := range implicitConflicts {
+		// A directory is listed in the report if and only if it was listed
+		// explictly in an SDF, meaning there is no conflict.
+		if _, ok := report.Entries[path]; !ok {
+			logf(`Warning: Attempted to extract path "%s" with different `+
```

Comment:
This may be shortened a bit:

"Warning: Path %q has diverging modes in different packages. Please report."